OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -395,7 +395,8 @@ public String getOAuthClientId() {
     }
 
     public String getOrganizationCode() {
-        return getString(ORGANIZATION_CODE, "");
+        String orgCode = getString(ORGANIZATION_CODE, "");
+        return orgCode.length() > 0 ? orgCode : null;
```

Comment:
Just using the `getString(ORGANIZATION_CODE)` method (without passing the default value parameter) should get you a `null` if the field isn't available.